include/js/general.js

```diff
@@ -91,6 +91,15 @@ GlobalVariable_getVariable('Application_Menu_Direction', 'Horizontal', (typeof g
 }, function (error) {
 	Application_Menu_Direction = 'Horizontal'; // set default value on error
 });
+
```

Comment:
this block is unnecessary, the code is already evaluating this global variable right above this block that you added and also the way you did it you have a different default value for the error.

on line 95 you set the default value to Vertical, then you call the getvariable with a default value of Vertical, but if that call fails, you set the default value to Horizontal on line 100. The code above sets the default value to Horizontal always which is how it should be.